fix makefile
Index: fcoe-utils/Makefile.am
===================================================================
--- fcoe-utils.orig/Makefile.am	2011-12-20 13:34:54.013964563 +0530
+++ fcoe-utils/Makefile.am	2011-12-20 13:40:43.549405953 +0530
@@ -60,7 +60,7 @@
 	if [ ! -f ${DESTDIR}${fcoe_configdir}/config ] ; then \
 		cp ${srcdir}/etc/config ${DESTDIR}${fcoe_configdir}/config; \
 	fi; \
-	if [ ! -d "${BASH_COMPLETION_DIR}" ] ; then \
-		mkdir ${BASH_COMPLETION_DIR}; \
+	if [ ! -d "${DESTDIR}${BASH_COMPLETION_DIR}" ] ; then \
+		mkdir ${DESTDIR}${BASH_COMPLETION_DIR}; \
 	fi; \
-	cp -f ${srcdir}/contrib/bash_completion/* ${BASH_COMPLETION_DIR}
+	cp -f ${srcdir}/contrib/bash_completion/* ${DESTDIR}${BASH_COMPLETION_DIR}
